sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -48,7 +49,7 @@
  *
  * @param <T> the type of the value
  */
-public abstract class WindowedValue<T> {
+public abstract class WindowedValue<T>  implements Serializable {
```

Comment:
I believe the need to force things to be `Serializable` may indicate we should to be using coders to move things over the wire in `byte[]` form. 
This technique is widely used throughout the `spark-runner` module by utilizing the
`CoderHelpers.toByteArray()` and `CoderHelpers.fromByteArray()` methods which are then passed in to Spark's `map()` transformations.